src/java.base/share/classes/jdk/internal/event/ThrowableTracer.java

```diff
@@ -22,47 +22,50 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
+package jdk.internal.event;
 
-package jdk.jfr.internal.instrument;
-
+import java.lang.reflect.Field;
 import java.util.concurrent.atomic.AtomicLong;
 
-import jdk.jfr.events.EventConfigurations;
-import jdk.jfr.events.ErrorThrownEvent;
-import jdk.jfr.events.ExceptionThrownEvent;
-import jdk.jfr.internal.event.EventConfiguration;
-
+/**
+ * Helper class for exception events.
+ */
 public final class ThrowableTracer {
 
     private static final AtomicLong numThrowables = new AtomicLong();
 
-    public static void traceError(Error e, String message) {
-        if (e instanceof OutOfMemoryError) {
+    public static void enable() throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException {
```

Comment:
SecurityException and IllegaArgumentException are unchecked so don't need them in the throws declaration.

Reply:
Will fix.